csrankings-p.csv

```diff
@@ -1033,6 +1033,7 @@ Prashant J. Shenoy,University of Massachusetts Amherst,https://people.cs.umass.e
 Prashant Jayaprakash Nair,University of British Columbia,https://prashantnair.bitbucket.io,IweLDwwAAAAJ
 Prashant Krishnamurthy,University of Pittsburgh,http://www.sci.pitt.edu/faculty-and-research/faculty-directory/prashant-krishnamurthy,F9qypa0AAAAJ
 Prashant V. Krishnamurthy,University of Pittsburgh,http://www.sci.pitt.edu/faculty-and-research/faculty-directory/prashant-krishnamurthy,F9qypa0AAAAJ
+Prashant Khanduri,Wayne State University,https://sites.google.com/view/khanduri-prashant/home?authuser=0,NOSCHOLARPAGE

```

Comment:
No indication of new affiliation on website. Please clarify.

Reply:
I am sorry for the late reply.

Prashant is going to join the computer science department at WSU in this coming fall. I noticed that CS ranking updates quarterly, I was wondering when CS ranking updates next time in Oct., He should have been working for WSU. The Chair of our department has approved to submit this request.

Please feel free to skip my request if you think it is too early.

Thanks.

Reply:
I need _some_ documentation for these faculty appointments.

Reply:
Prashant has updated his affiliation at https://sites.google.com/view/khanduri-prashant/home . Please check. Thanks.